ui/app/styles/core/message.scss

```diff
@@ -78,6 +78,10 @@
     background: $red-010;
     border: 1px solid $red-050;
 
+    .message-body {
+      color: $red-700; // replaced hard-coded #8e2c37 with color var for consistency.
```

Comment:
I don't think we need to keep this message in the code -- would have been a great PR comment though!

Reply:
Roger that. I'll go ahead and remove those comments and add to the PR.